src/js/views/globals/app-frame.scss

```diff
@@ -17,9 +17,25 @@
   }
 }
 
+@keyframes reveal-content {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
 .app-frame__content {
   min-width: 440px;
+  opacity: 0;
   overflow-y: auto;
+
+  &:not(:empty) {
+    animation: 0.3s cubic-bezier(0.165, 0.84, 0.44, 1) 0s reveal-content forwards;
+    background-color: $white;
```

Comment:
making the app-root the background color requires the content to have a white background when not empty.  All other elements of the app frame include a background style